raiden/connection_manager.py

```diff
@@ -166,7 +166,7 @@ def connect(
                 views.state_from_raiden(self.raiden), self.registry_address, self.token_address
             )
 
-            if not qty_network_channels:
+            if qty_network_channels == 0:
```

Comment:
Not a bugfix. Making the check explicit so the reader and the code knows what to expect for `qty_network_channels`.